mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping

```diff
@@ -8,28 +8,38 @@ CLASS net/minecraft/class_1293 net/minecraft/entity/effect/StatusEffectInstance
 	FIELD field_5895 duration I
 	FIELD field_5896 type Lnet/minecraft/class_1291;
 	FIELD field_5897 LOGGER Lorg/apache/logging/log4j/Logger;
+	METHOD <init> (Lnet/minecraft/class_1291;)V
+		ARG 1 type
 	METHOD <init> (Lnet/minecraft/class_1291;I)V
 		ARG 1 type
+		ARG 2 duration
 	METHOD <init> (Lnet/minecraft/class_1291;II)V
 		ARG 1 type
 		ARG 2 duration
+		ARG 3 amplifier
 	METHOD <init> (Lnet/minecraft/class_1291;IIZZ)V
-		ARG 1 effect
+		ARG 1 type
 		ARG 2 duration
 		ARG 3 amplifier
-		ARG 5 showParticles
+		ARG 4 ambient
+		ARG 5 visible
```

Comment:
The field is called `showParticles`, so this should be `showParticles` as well (it's called `ShowParticles` in the NBT).

Reply:
i think this is a vararg ctor in which this param is passed into both show particle and show in inv screen params of another constructor.

Reply:
Ah, right. Should've looked in the ctor body.